README.md

```diff
@@ -8,3 +8,26 @@ Utilities for maintaining the [Fedora Commons repository](http://github.com/fcre
     * fedora:mimeType => ebucore:hasMimeType
     * premis:hasOriginalName => ebucore:filename
 
+Building
+--------
+
+To build the JAR file
+
+``` sh
+mvn package
+```
+
+Running
+-------
+
+To run the migration utility, execute the JAR file and provide the `fcrepo.home` system property to set the directory that contains Fedora 4's `fcrepo4-data` directory.
```

Comment:
Actually, I find that in order for the utility to work, `fcrepo.home` must be set to the `fcrepo4-data` directory, not just `the directory that contains Fedora 4's fcrepo4-data directory`.